src/bitcoin-tx.cpp

```diff
@@ -433,13 +433,16 @@ static void MutateTxAddOutData(CMutableTransaction& tx, const std::string& strIn
     if (pos==0)
         throw std::runtime_error("TX output value not specified");
 
-    if (pos != std::string::npos) {
+    if (pos == std::string::npos) {
```

Comment:
nit: Preserving the original condition, i.e., switching branches `if` and `else`, makes diff smaller.

Reply:
Double negation makes code harder to read

Reply:
Since `npos` is already a negation, in this case the `else` would be triple-negated